src/Catalyst.Node.POA.CE/Program.cs

```diff
@@ -160,17 +156,17 @@ private static async Task CustomBootLogic(Kernel kernel)
             foreach (var module in modulesToRegister) containerBuilder.RegisterModule(module);
         }
 
-        public static int Main(string[] args)
+        public static async Task<int> Main(string[] args)
         {
             // Parse the arguments.
-            Parser.Default
+            var result = await Parser.Default
                .ParseArguments<Options>(args)
-               .WithParsed(async o => await RunAsync(o).ConfigureAwait(false));
+               .MapResult(async options => await RunAsync(options), response => Task.FromResult(1));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4699939)